pkg/kubectl/cmd/create/create_job.go

```diff
@@ -240,4 +242,10 @@ func (o *CreateJobOptions) createJobFromCronJob(cronJob *batchv1beta1.CronJob) *
 		},
 		Spec: cronJob.Spec.JobTemplate.Spec,
 	}
+
+	job.SetOwnerReferences([]metav1.OwnerReference{
+		*metav1.NewControllerRef(cronJob, apps.SchemeGroupVersion.WithKind("CronJob")),
```

Comment:
Set it directly in `ObjectMeta` similarly how it's done in https://github.com/kubernetes/kubernetes/blob/2d956389dc57fb1715d1832b2d22c042de59bd1d/pkg/controller/cronjob/utils.go#L158-L165